Samples/SDKFeatures/SDKFeatures/MHVMoreFeatures.m

```diff
@@ -90,9 +90,62 @@ -(void)getServiceDefinition
         
         [MHVUIAlert showInformationalMessage:output];
         
-        [m_controller.statusLabel clearStatus];
+        [self.listController.statusLabel clearStatus];
         
     }] start];  // NOTE: Make sure you always call start
+#else
+    id<MHVSodaConnectionProtocol> connection = [[MHVConnectionFactory current] getOrCreateSodaConnectionWithConfiguration:[MHVFeaturesConfiguration configuration]];
+    
+    [connection.platformClient getServiceDefinitionWithCompletion:^(MHVServiceDefinition * _Nullable serviceDefinition, NSError * _Nullable error)
+    {
+        if (error)
+        {
+            [MHVUIAlert showInformationalMessage:error.localizedDescription];
+        }
+        else
+        {
+            MHVConfigurationEntry* configEntry = [serviceDefinition.platform.config objectAtIndex:0];
+            MHVConfigurationEntry* configEntry2 = [serviceDefinition.platform.config objectAtIndex:1];
+            NSMutableString* output = [[NSMutableString alloc] init];
+            
+            [output appendLines:17, @"Some data from ServiceDefinition",
+             @"[PlatformUrl]", serviceDefinition.platform.url,
+             @"[PlatformVersion]", serviceDefinition.platform.version,
+             @"[ShellUrl]", serviceDefinition.shell.url,
+             @"[ShellRedirect]", serviceDefinition.shell.redirectUrl,
+             @"[Example Config Entries]",
+             configEntry.key, @"==", configEntry.value, @"==========",
+             configEntry2.key, @"==", configEntry2.value];
+            
+            [MHVUIAlert showInformationalMessage:output];
+        }
+        
+        
+        [self.listController.statusLabel clearStatus];
```

Comment:
need to dispatch to main thread.  Should we have a setting on the SDK to do the completions on the main thread?  Could affect performance when set, but for a simple app it would also ease development using the SDK.